tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -836,12 +836,14 @@ Polymer({
       this.getBoundingClientRect().right -
       inputControl.getBoundingClientRect().right + 30) + 'px';
     this.showDeleteValueButton = true;
-
+    inputControl.style.border = '1px solid #ffeb3b';
```

Comment:
though there are enough examples of this, can we just use the CSS pseudo-selector?
i.e.,
```css
paper-input:focus {
  border: 1px solid #ffeb3b;
}
```

I want styling logic/code to be yanked away from JS into CSS as much as possible.

Reply:
i also want this in CSS, but iron-autogrow-textarea :focus selector doesn't work as expected. even when the item is focused by user input (clicking, typing), that css selector isn't chosen. must have to do with its design (contains a div which contains a textarea). wasn't aware of a way to get the selector on that element based on properties of a grandchild element, so moved to JS code.

do you have any ideas of how it can be done just in CSS in this case?

Reply:
```css
iron-autogrow-textarea[focused] {
  border: 1px solid #ffeb3b;
}
```

seems to work

Reply:
thanks that works! i was trying ":focus" in the specifier and that wasn't being picked up.